death_note.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "sync"
+
+type DeathNote struct {
+	paramSet map[string]bool
```

Comment:
```suggestion
	notes map[string]bool
```

I think it would be beneficial to stick with the original mythology, then it would be easier to understand the analogy. Then get/read would become `Note(note string)`/`Notes() []string`